src/hooks/login/useLedgerLogin.ts

```diff
@@ -59,8 +58,8 @@ export function useLedgerLogin({
   callbackRoute,
   token,
   addressesPerPage = defaultAddressesPerPage,
-  redirectAfterLogin = false
 }: UseLedgerLoginPropsType): LedgerLoginHookReturnType {
+  const redirectAfterLogin = Boolean(callbackRoute);
```

Comment:
do we use this for anything here? If this is passed down, let's just pass callbackRoute until it's actually used